examples/api/test/material/color_scheme/dynamic_content_color.0_test.dart

```diff
@@ -0,0 +1,156 @@
+        expect(find.text('error'), findsExactly(2));
+        expect(find.text('onError'), findsExactly(2));
+        expect(find.text('errorContainer'), findsExactly(2));
+        expect(find.text('onErrorContainer'), findsExactly(2));
+        expect(find.text('surface'), findsExactly(2));
+        expect(find.text('onSurface'), findsExactly(2));
+        expect(find.text('onSurfaceVariant'), findsExactly(2));
+        expect(find.text('outline'), findsExactly(2));
+        expect(find.text('shadow'), findsExactly(2));
+        expect(find.text('inverseSurface'), findsExactly(2));
+        expect(find.text('onInverseSurface'), findsExactly(2));
+        expect(find.text('inversePrimary'), findsExactly(2));
+
+        ThemeData themeData = Theme.of(
+          tester.element(find.byType(Scaffold)),
+        );
+
+        expect(themeData.colorScheme.primary, const Color(0xff575992));
+        expect(themeData.colorScheme.secondary, const Color(0xff5d5c72));
+
+        await tester.tap(find.byType(Switch));
+        await tester.pump();
+
+        // Loads the images.
+        await tester.runAsync(() => Future<void>.delayed(Duration.zero));
+        await tester.runAsync(() => Future<void>.delayed(Duration.zero));
+        await tester.pump();
+        await tester.pump(kThemeChangeDuration);
+
+        themeData = Theme.of(tester.element(find.byType(Scaffold)));
+
+        expect(themeData.primaryColor, const Color(0xff131318));
+        expect(themeData.colorScheme.secondary, const Color(0xffc6c4dd));
+      },
+      createHttpClient: (SecurityContext? securityContext) => _FakeClient(),
+    );
+  });
+}
+
+class _FakeClient extends Fake implements HttpClient {
+  @override
+  Future<HttpClientRequest> getUrl(Uri url) async {
+    return _FakeHttpClientRequest();
+  }
+
+  @override
+  bool autoUncompress = true;
+}
+
+class _FakeHttpClientRequest extends Fake implements HttpClientRequest {
+  @override
+  HttpHeaders get headers => _FakeHttpHeaders();
+
+  @override
+  Future<HttpClientResponse> close() async {
+    return _FakeHttpClientResponse();
+  }
+}
+
+class _FakeHttpHeaders extends Fake implements HttpHeaders {}
+
+class _FakeHttpClientResponse extends Fake implements HttpClientResponse {
+  @override
+  HttpClientResponseCompressionState get compressionState => HttpClientResponseCompressionState.notCompressed;
+
+  @override
+  int get contentLength => _blueSquarePng.length;
+
+  @override
+  int get statusCode => 200;
+
+  @override
+  Future<E> drain<E>([E? futureValue]) {
+    return Future<E>.value(futureValue as E);
+  }
+
+  @override
+  StreamSubscription<List<int>> listen(void Function(List<int> event)? onData,
+      {Function? onError, void Function()? onDone, bool? cancelOnError}) {
+          return Stream<List<int>>.value(_blueSquarePng).listen(
+            onData,
+            onDone: onDone,
+            onError: onError,
+            cancelOnError: cancelOnError,
+          );
+
+  }
```

Comment:
```suggestion
  StreamSubscription<List<int>> listen(
    void Function(List<int> event)? onData, {
    Function? onError,
    void Function()? onDone,
    bool? cancelOnError,
  }) {
    return Stream<List<int>>.value(_blueSquarePng).listen(
      onData,
      onDone: onDone,
      onError: onError,
      cancelOnError: cancelOnError,
    );
  }
```

Reply:
That's way better, I changed it in [format: Add better comment for the runAsyncs and better formatting](https://github.com/flutter/flutter/pull/158309/commits/90d28317d442abbfa4cb8c36f904a2d63f42a47b)